admin-guides/saml-sso.mdx

```diff
@@ -33,3 +33,30 @@ Now to the steps:
 <Note>
   After saving the SAML SSO settings, you will be logged out and will need to log back in. You have to type in your email address to initiate the SAML SSO flow.
 </Note>
+
+## Notes
+
+### Okta Group Assignments
+
+When assigning Groups to the Okta App used on Koala's integration, make sure that the users assigned from that group are shown as "Group" type instead of "Individual". This _inconsistent scenario_ could happen if a user that belongs to the group was previously assigned as an "individual", making the login process fail.
+
+![](/images/admin-guides/okta-group-assignment.png)
+
+To fix this, remove all previously individually assigned users and then assign the Group you want to give access to. After that, all users in that group will be assigned to App and the type should be shown as "Group".
+
+### Okta Attribute Statements
+
+To correctly create and activate new users on Koala using Okta's info, you should configure the [Attribute Statements](https://help.okta.com/oie/en-us/content/topics/apps/aiw-saml-reference.htm?cshid=csh-attribute-statements-saml) on the Okta App used on integration following these steps:
+
+1. Click on the App settings and go to the "General" tab;
+2. Click on the **Edit** button on the **SAML Settings** section;
+3. Click on **Next** button to go to **2. Configure SAML** step;
+4. Locate the **Attribute Statement** section and add the following fields:
```

Comment:
We need to use `name` too

Reply:
There is no "full name" option on Okta:
<img width="352" alt="image" src="https://github.com/user-attachments/assets/a8cf404c-e6db-46c6-8bb0-d406d1d0711e" />

Reply:
I'll ship as is. If I found a way to map the full name, I could update it later.

Reply:
fullName or displayName? is it not present?